pkg/roachpb/data.go

```diff
@@ -2488,7 +2488,9 @@ var _ = (SequencedWriteBySeq{}).Find
 
 func init() {
 	// Inject the format dependency into the enginepb package.
-	enginepb.FormatBytesAsKey = func(k []byte) string { return Key(k).String() }
+	enginepb.FormatBytesAsKey = func(k []byte) redact.RedactableString {
+		return redact.Sprint(Key(k).String())
```

Comment:
Consider not calling the `String()` method here. The `Key` type has a `Format()` method which `redact.Sprint` can already use, and if we ever add a `SafeFormat()`  method to `Key` everything will compose nicely.